docs/guide/topics/error_handling.rst

```diff
@@ -0,0 +1,361 @@
+            await ctx.send(f'Please mention the person to ban using `{ctx.clean_prefix}ban @member`')
+
+*2. Then, in our error handler, we use ``getattr()`` to check for the custom attribute we just set:*
+
+..
+    Note, I couldn't get to reference getattr() to the python docs... >_<
+
+.. code-block:: python3
+
+    @bot.listen('on_command_error')
+    async def global_error_handler(ctx, error):
+        """ Global error handler that handles errors that aren't maked as ignored"""
+
+        # Check if `ctx.ignore_error` is True
+        if getattr(ctx, 'ignore_error', False) is True:
+            return  # we ignore the error
+
+        # Simpler way to handle CommandInvokeError using getattr
+        error = getattr(error, 'original', error)
+
+        # Then we continue with our error handling as normal...
+        print(f"Ignoring unhandled exception in command {ctx.command!r}")
+        traceback.print_exception(type(error), error, error.__traceback__)
+
+In :ref:`Events <discord-api-events>`
+-------------------------------------
+
+If an error ocurrs in an event, it does not go to any command error handler, as it is not a command. These errors are instead
+propagated to :meth:`~Client.on_error`, and can be retrieved with a standard call to :func:`sys.exc_info`. 
+
+**Example**
+
+.. code-block:: python3
+
+    import sys
+    import traceback
+
+    @client.event
+    async def on_error(event, *args, **kwargs):
+        """ Handles errors in events. """
+
+        # Firs we get the information of the error:
+        err_type, err_value, err_traceback = sys.exc_info()
+
+        # Then we can use it to generate a traceback text:
+        err_lines = traceback.format_exception(err_type, err_value, err_traceback)
+        err_text = ''.join(err_lines)
+
+        # Then we can do whatever with these, for example, printing them to console,
+        # or even sending them to some log channel to keep track of them!
+        print(f"Ignoring unhandled exception in event {event!r}\n{err_text}")
+
+.. note::
+
+    ``on_error`` will only be dispatched to :meth:`Client.event`.
+
+    It will not be received by :meth:`Client.wait_for`, or, if used, :ref:`ext_commands_api_bot` listeners such as
+    :meth:`~ext.commands.Bot.listen` or :meth:`~ext.commands.Cog.listener`.
+
+In :ref:`Interactions <interactions_api>`
+-----------------------------------------
+
+In :ref:`Views <discord_ui_kit>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Handling errors in a :class:`~.ui.View` is as simple as overriding it's :meth:`~.ui.View.on_error`.
+
+**Example**
+
+.. code-block:: python3
+
+    class ErrorHandledView(discord.ui.View):
+
+        async def on_error(self, error, item, interaction):
+            """ Handles errors for this view. """
+            ...
+
+In :ref:`Application Commands <discord_app_commands>`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To handle errors in a :class:`~.app_commands.CommandTree` there's two ways of doing so:
```

Comment:
```suggestion
There are two ways of handling errors in a :class:`~.app_commands.CommandTree`:
```
This is grammatically incorrect. I was originally thinking you should change it to `To handle errors in a CommandTree there are two ways of doing so:` but that sounds weird and wrong. Perhaps you can use this or `Handling errors in a CommandTree requires you to use one of the following:` or `To handle errors in a CommandTree there are two ways:` or something similar.